gui/mozregui/ui/global_prefs.ui

```diff
@@ -33,6 +33,23 @@
       <widget class="DirectorySelectWidget" name="persist" native="true"/>
      </item>
      <item row="1" column="0">
+         <widget class="QLabel" name="label">
+             <property name="toolTip">
+                 <string>The limit size of persist directory.</string>
+             </property>
+             <property name="text">
+                 <string>persist size limit:</string>
+             </property>
+         </widget>
+     </item>
+     <item row="1" column="1">
+         <widget class="QDoubleSpinBox" name="persist_size_limit">
+             <property name="minimum">
+                 <double>1.0000</double>
```

Comment:
Please put the minimum to 0.0 (it would mean deactivated), and the max to more than 99.99 (999.99 should be good).